### ISSUE TITLE
[❔]About using the "Context Menu - Selected" command from the VSCode command palette

### ISSUE THREAD

**Issue by claudio-salvio** [electrotype/vscode-windows-explorer-context-menu#6]
Hello @electrotype

I tried to run "Context Menu - Selected" from the VSCode command palette but nothing happens.
![image](https://user-images.githubusercontent.com/66575834/159085556-8576ba8c-ca10-4feb-9f5b-262625ed71e0.png)

I thought that doing so would open the "windows context menu" from the file shown in the editor but I was wrong.
Is there any way to do it?

My ultimate goal was to associate a shortcut key (e.g. CTRL+APPS) to the "Context Menu - Selected" command so that I could execute it directly.

🙏Thanks for making this useful extension available.

Best regards,
Claudio

**Comment by electrotype** [comment]
Hello @claudio-salvio ,

I just published a `3.2.0` version to address this. There are some limitations, but I think it does provide the feature that you are looking for.

- You need to use the "`☰ Context Menu`" command from the command palette to target the file _open in the active editor_, not "`☰ Context Menu - Selected`".

- The command "`☰ Context Menu - Root`" also works, as long as you only have _one_ workspace open in your VSCode instance.

- The command "`☰ Context Menu - Selected`" (that targets selected files in the `Explorer` view) doesn't work and currently [_cannot_  work](https://github.com/microsoft/vscode/issues/3553).

Let me know if you find any issues!

**Comment by claudio-salvio** [comment]
Hello @electrotype,

Thank you very much for adding this functionality so quickly !
It works perfectly well.

I generated a keybinding from `ContextMenu` to `SHIFT+F10` when `editorTextFocus` that allows me to access the **Windows Context Menu** directly. 

I had originally planned to generate the keybinding to some combination with the `CONTEXTMENU` key.
Unfortunately none of them worked, I tried `ALT+CONTEXTMENU`, `CTRL+CONTEXTMENU`, `SHIFT+CONTEXTMENU` and in all cases it still opens the **VSCode context menu** and not the **Windows Context Menu**.
I suspect that VSCode reserves the use of the `CONTEXTMENU` (aka `APPS`) key somehow because in the list of keybindings there is no keybinding that uses it.

Best regards,
Claudio Salvio
